src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,58 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    category: {
+      id: null,
+      name: '',
+    },
+    categoryList: [],
+    loading: false,
+    errorMessage: '',
+    doneMessage: '',
+  },
+  mutations: {
+    clearMessage(state) {
+      state.errorMessage = '';
+      state.doneMessage = '';
+    },
+    updateValue(state, { name, value }) {
+      state.user = { ...state.user, [name]: value };
+    },
+    // カテゴリーリスト取得
+    doneGetAllCategories(state, { categories }) {
+      state.categoryList = categories;
+      state.loading = false;
+    },
+    failRequest(state, { message }) {
+      state.errorMessage = message;
+      state.loading = false;
+    },
+  },
+  actions: {
+    clearMessage({ commit }) {
+      commit('clearMessage');
+    },
+    updateValue({ commit }, target) {
+      commit('updateValue', target);
+    },
+    // カテゴリー全件取得
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(response => {
+        if (response.data.code === 0) throw new Error(response.data.message);
+
+        const categories = response.data.categories.map(data => ({
+          id: data.id,
+          name: data.name,
+        }));
+        commit('doneGetAllCategories', { categories });
```

Comment:
[imo]

渡す変数が1つなのであれば、そのまま渡しても良いかなと思います。
mutations の方の修正も必要になるかと思います。

```suggestion
        commit('doneGetAllCategories', categories);
```

Reply:
mutationsとactionsにて上記{}を削除しました